domains/ycat.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Chalk01",
+    "email": "c.superchayu@gmail.com"
+  }
```

Comment:
Missing a comma here

Reply:
Sorry, I'm new to github but what am I supposed to do now?

Reply:
> Sorry, I'm new to github but what am I supposed to do now?

edit the file and add a `,` after the `}`

Reply:
```suggestion
  },
```

Reply:
I've done it?

Reply:
wait.